packaging/convert2rhel.spec

```
@@ -124,6 +124,27 @@ install -m 0600 config/convert2rhel.ini %{buildroot}%{_sysconfdir}/convert2rhel.
 %attr(0644,root,root) %{_mandir}/man8/%{name}.8*
 
 %changelog
+* Mon Feb 20 2023 Eric Gustavsson <egustavs@redhat.com> 1.2
+- Remove python 2.6 and RHEL 6 conversion capability
```

```suggestion
- Remove RHEL 6 conversion capability
```
Regarding Python 2.6, it's more of a removing/losing compatibility rather than capability. I think that leaving out the Python version is fine. If you want to keep have it there then it might be better to create a new bullet.